pandas/tests/extension/test_arrow.py

```diff
@@ -3203,6 +3203,18 @@ def test_pow_missing_operand():
     tm.assert_series_equal(result, expected)
 
 
+@pytest.mark.skipif(
+    pa_version_under11p0, reason="Decimal128 to string cast implemented in pyarrow 11"
```

Comment:
So what happens with pyarrow < 11 right now - is there a different error or are results nonsensical?

Reply:
NotImplementedError. string to decimal cast only added in pyarrow 11: https://github.com/apache/arrow/issues/33002